R/pruneScores.R

```diff
@@ -54,25 +53,31 @@
 #' # Running the SingleR() example.
 #' example(SingleR, echo=FALSE)
 #'
-#' summary(pruneScores(pred$scores))
+#' summary(pruneScores(pred))
 #'
 #' # Less stringent:
-#' summary(pruneScores(pred$scores, min.diff.med=0))
-#' summary(pruneScores(pred$scores, nmads=5))
+#' summary(pruneScores(pred, min.diff.med=0))
+#' summary(pruneScores(pred, nmads=5))
+#' 
+#' # More stringent:
+#' summary(pruneScores(pred, min.diff.med=0.1))
+#' summary(pruneScores(pred, nmads=2))
+#' summary(pruneScores(pred, min.diff.next=0.1))
 #' 
 #' @export
 #' @importFrom DelayedMatrixStats rowMedians 
 #' @importFrom DelayedArray DelayedArray rowMaxs
 #' @importFrom stats median mad
-pruneScores <- function(scores, min.diff.med = 0.05, nmads=3, min.diff.next = 0) {
-    if (is(scores, "DataFrame")){
-      scores <- scores$scores
-    }
+pruneScores <- function(results, min.diff.med = 0.05, nmads=3, min.diff.next = 0) {
```

Comment:
Make sure that `results` is consistent with the argument names for similar inputs to all the visualization functions.

Reply:
Done (duplicated comments for same thing)